woocommerce/woocommerce-gutenberg-products-block
# Move cart and mini-cart blocks into their own folders

Fixes #5686


We're recently taken the checkout block outside the `js/assets/blocks/cart-checkout` folder  ([PR](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5594)) into it's own `checkout` folder. This PR does the same for the cart and the mini-cart blocks. This is clearer as each block now lives under its' own folder, and any shared code across two or more blocks lives in `js/assets/blocks/shared`. This also simplifies the webpack config as we don't need to add `customDir` options for the `cart-checkout` folder, instead it now matches the regular path pattern of all other blocks.

### Manual Testing

How to test the changes in this Pull Request:

1. Smoke test the Cart Block to make sure it works as expected
2. Smoke test the Mini-cart block to make sure it works as expected
3. Smoke. test the Checkout block to make sure it works as expected
